change_log/next/FE-1728-radio-button.yml

```diff
@@ -0,0 +1,3 @@
+New Features: "Wrapped some components in the withValidations component. You can still export the components without the validation HOC. (Components: RadioButton, RadioButtonGroup)"
```

Comment:
I think we said that `You can still export the components without the validation HOC` was unnecessary. We should just expose a single component as it works exactly the same.

Reply:
description changed